courses/machine_learning/deepdive2/structured/labs/4b_keras_dnn_babyweight.ipynb

```diff
@@ -0,0 +1,574 @@
+{
```

Comment:
<p>the description of DEFAULTS took me a second.</p><p> </p><p>Should it be "DEFAULTS is a list of length equal to the length CSV_COLUMNS". Each element of DEFAULTS is a list itself with the default value for the respective CSV column.</p><p> </p><p>If re-written, change in solution notebook to match.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/training-data-analyst/pull/672/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='9'/>

Reply:
<p>Ya, I changed it, it was a little convoluted sounding, thanks!</p>